ecommerceproj/theretailerapp/views.py

```diff
@@ -315,11 +377,17 @@ def filter_orders(request):
     else:
         return HttpResponseRedirect('/customer/login')
 
-def increase_basket_item_quantity(request,product_id):
+def increase_basket_item_quantity(request:RequestType,product_id:uuid):
+    """
+    Increases the basket item quantity by one. Either in db or session.
+    """
     add_product_to_basket(request,product_id)
     return HttpResponseRedirect('/basket')
 
-def decrease_basket_item_quantity(request,product_id):
+def decrease_basket_item_quantity(request:RequestType,product_id:uuid):
```

Comment:
`product_id:uuid` ??

`uuid` is not a data type. Think of type hinting as data type declaration in strongly typed languages like C, Java etc.